src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.31.1"
+__version__ = "0.32.0"
```

Comment:
```suggestion
__version__ = "0.31.2"
```
Please fix the conflicts on your branch as well

Reply:
Done. Versin in 0.32.0